Extract/Extract.ipynb

```diff
@@ -0,0 +1,264 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import re\n",
+    "import pandas as pd\n",
+    "import csv\n",
+    "import string\n",
+    "from tkinter import *\n",
+    "from tkinter.filedialog import askopenfilename\n",
+    "import time\n",
+    "import os\n",
+    "import urllib\n",
+    "from pathlib import Path\n",
+    "# Uncomment the next two lines if you want to see expanded output\n",
+    "#pd.options.display.max_colwidth\n",
+    "#pd.set_option('display.max_colwidth', -1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Select the text you want to upload\n",
+    "This will open a file picker in a new window"
```

Comment:
Period at the end of the sentence is missing. Also, does the file dialog still open behind the the current window? If so, there should be a note about that here.

Reply:
Removed tkinter and changed the text to instruct users to paste the file path into the code.